src/main.c

```diff
@@ -132,34 +143,22 @@ void Process_Input(BMS_INPUT_T* bms_input) {
         Board_CAN_ProcessInput(bms_input, &bms_output); // CAN has precedence over console
         Board_LTC6804_ProcessInputs(&pack_status, &bms_state);
     }
-
-    bms_input->contactors_closed = Board_Contactors_Closed();
     bms_input->msTicks = msTicks;
+#ifdef FSAE_DRIVERS
+    bms_input->contactors_closed = Fsae_Fault_Pin_Get();
+#endif
 }
 
 void Process_Output(BMS_INPUT_T* bms_input, BMS_OUTPUT_T* bms_output, BMS_STATE_T * bms_state) {
     // If SSM changed state, output appropriate visual indicators
     // Carry out appropriate hardware output requests (CAN messages, charger requests, etc.)
-#ifdef FSAE_DRIVERS
```

Comment:
@skoppula @jatron why was the contactor logic FSAE-guarded before? Does EVT code not want to set the state of the fault pin to match the output?